odoo/modules/module.py

```diff
@@ -97,12 +97,13 @@ def __init__(self, pydep):
 # matches field definitions like
 #     partner_id: base.ResPartner = fields.Many2one
```

Comment:
We should not handle this anymore ("field_type" in regex). Because the single place for typing is "type_param".